docs/sources/fundamentals/architecture/rings.md

```diff
@@ -91,3 +94,7 @@ despite the compactor target being on multiple instances.
 ## About the ruler ring
 
 The ruler ring is used to determine which rulers evaluate which rule groups.
+
+## About the index gateway ring
+
+The index gateway ring is used to determine which gateway is responsible for which tenant's indices when queried by rulers or queriers.
```

Comment:
Indexes refers to more than one index.  Indices are for ranges, like an array index.
```suggestion
The index gateway ring is used to determine which gateway is responsible for which tenant's indexes when queried by rulers or queriers.
```